apps/asap-server/src/migrations/1636027226-move-repurposed-fields.ts

```diff
@@ -39,8 +50,9 @@ async function migrateTeamFields() {
   await applyToAllItemsInCollection<RestTeam>(
     'teams',
     async (team, squidexClient) => {
+      const { skills } = (team as unknown as Rest<{ skills: string[] }>).data;
```

Comment:
i dont think you should change the logic of the old migrations. I see you're having to cast some types (i would try to do as little as possible in that area too) but probably leave the logic as is, if possible

Reply:
done